dynamic_programming/levenshtein_distance.nim

```diff
@@ -0,0 +1,72 @@
+## Levenshtein distance
+
+import tables
+
+
+func initSubsolutions(lenA: Natural, lenB: Natural): Table[(Natural, Natural), Natural] =
+  for indA in 0..lenA:
+    result[(Natural(indA), Natural(0))] = Natural(indA)
+
+  for indB in 1..lenB:
+    result[(Natural(0), Natural(indB))] = Natural(indB)
```

Comment:
```suggestion
func initSubsolutions(lenA, lenB: Natural): Table[(Natural, Natural), Natural] =
  for indA in 0.Natural..lenA:
    result[(indA, 0.Natural)] = indA

  for indB in 1.Natural..lenB:
    result[(0.Natural, indB)] = indB
```
Just a bit less cluttering.

Reply:
My intention was to use `toKey`, cf. 093c244e9ea308a7f84066466b3f16e3fbe5dd65. Thanks for spotting that.